api/v1alpha4/cluster_types.go

```diff
@@ -381,7 +381,7 @@ const (
 )
 
 func (f ClusterIPFamily) String() string {
-	return [...]string{"InvalidIPFamily", "IPv4IPFamily", "IPv6IPFamily", "DualStackIPFamily"}[f]
+	return [...]string{"Invalid", "IPv4", "IPv6", "DualStack"}[f]
```

Comment:
I think Fabrizio meant transforming the strings returned by the func in the variable code (or alternatively something with switch case, which is probably more stable)

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/6171#discussion_r816074839 :+1:

Reply:
This is a breaking change.
I was thinking to drop the suffix only when setting the builtin variable

Reply:
I'll continue above